docs/usage/pages.rst

```diff
@@ -50,4 +50,12 @@ All ``breadcrumbs`` fields have the following processor enabled:
 
 .. autofunction:: zyte_common_items.processors.breadcrumbs_processor
 
+All ``price`` fields have the following processor enabled:
```

Comment:
It seems the documentation should also mention the default imlementations for currency and currencyRaw fields (currenctly only metadata and url are mentioned).

Reply:
It's a different kind of default implementation though (which does nothing out of box, but does some magic if user defines other things), but it still needs to be documented.